pkg/controllers/vdb/sandbox_configmap.go

```diff
@@ -60,7 +60,7 @@ func (s *SandboxConfigMapManager) triggerSandboxController(ctx context.Context)
 		// This will ensure that we always set a new value
 		// for this annotation which will wake up the sandbox
 		// controller
-		vmeta.SandboxControllerTriggerID: triggerID,
+		vmeta.SandboxControllerUpgradeTriggerID: triggerID,
```

Comment:
Perhaps make this function more general so we can use it to trigger for upgrade as well as for unsandbox?